liquibase-core/src/main/java/liquibase/parser/core/yaml/YamlParser.java

```diff
@@ -4,11 +4,21 @@
 import liquibase.logging.Logger;
 import liquibase.parser.LiquibaseParser;
 import liquibase.resource.ResourceAccessor;
+import org.yaml.snakeyaml.LoaderOptions;
 
 public abstract class YamlParser implements LiquibaseParser {
 
     protected Logger log = Scope.getCurrentScope().getLog(getClass());
 
+    public static LoaderOptions createLoaderOptions() {
```

Comment:
How was it determined which defaults to use in the LoaderOptions setters? We've seen some issues around code point limit recently and although MAX_VALUE seems to make sense, I want to double-check with all the Devs out there. 

CC @asomov @MalloD12

Reply:
Older versions of SnakeYaml do not have the ability to set a code point limit, If for some reason if older version of snakeyaml is loaded then this would thrown an error. It is safe to catch NoSuchMethodError. We have a method that serves the purpose in SnakeYamlUtil class which will be releasing in 4.191.1
@filipelautert

Reply:
Hi @suryaaki2, are you basically saying we should be surrounding these loader option sets into a try/catch block? In case of an error, should be shown any particular message? 

Thanks,
Daniel.

Reply:
Updating `setCodePointLimitSafely()` call by using the one defined in SnakeYamlUtil.

Reply:
Thanks for making that change, @MalloD12!